src/node_jsvmapi_types.h

```diff
@@ -1,4 +1,4 @@
-#ifndef SRC_NODE_JSVMAPI_TYPES_H_
+﻿#ifndef SRC_NODE_JSVMAPI_TYPES_H_
```

Comment:
This is due to what looks like a bug in VS 2017 which always adds a BOM, even when it's not required. There are quite a few files with these marks so I propose we fix them all at once.